### PR TITLE
Add possibility to convert private boards

### PR DESCRIPTION
**Pull request by daMatz**
Added variables for Trello API key and token in case a private board should be converted.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,10 +4,19 @@ const sanitize = require("sanitize-filename");
...
+    const privateBoard = options.privateBoard;
+    const trelloApiKey = options.trelloApiKey;
+    const trelloApiToken = options.trelloApiToken;
+
     // https://developers.trello.com/reference#section-nested-cards-as-query-params
-    const trelloBoardCardsEndPoint =`https://api.trello.com/1/boards/${boardId}/?cards=visible`;
+    var trelloBoardCardsEndPoint = `https://api.trello.com/1/boards/${boardId}/?cards=visible`;
     // const trelloBoardLabels = `https://api.trello.com/1/boards/${boardId}/labels`
-    const trelloBoardLists = `https://api.trello.com/1/boards/${boardId}/lists`
+    var trelloBoardLists = `https://api.trello.com/1/boards/${boardId}/lists`;
```
Comment (pietrop) [inline index.js]:
```suggestion
    let trelloBoardLists = `https://api.trello.com/1/boards/${boardId}/lists`;
```

File: index.js [hunk 2]
```diff
@@ -4,10 +4,19 @@ const sanitize = require("sanitize-filename");
 
 function trelloToMarkdownFolders(options){
     const boardId = options.boardId;
+    const privateBoard = options.privateBoard;
+    const trelloApiKey = options.trelloApiKey;
+    const trelloApiToken = options.trelloApiToken;
+
     // https://developers.trello.com/reference#section-nested-cards-as-query-params
-    const trelloBoardCardsEndPoint =`https://api.trello.com/1/boards/${boardId}/?cards=visible`;
+    var trelloBoardCardsEndPoint = `https://api.trello.com/1/boards/${boardId}/?cards=visible`;
```
Comment (pietrop) [inline index.js]:
```suggestion
    let trelloBoardCardsEndPoint = `https://api.trello.com/1/boards/${boardId}/?cards=visible`;
```

**Comment by pietrop** [comment]
This is great, thanks!

**Comment by daMatz** [comment]
Thank you for the project :)

**Comment by pietrop** [comment]
I just published the new version to npm

**Pull request merged**